src/store/forecast.slice.ts

```diff
@@ -60,5 +60,5 @@ const forecastSlice = createSlice({
   },
 })
 
-export const { setForecast, setForecastError } = forecastSlice.actions;
+export const { setForecast, setForecastError, setForecastLoading } = forecastSlice.actions;
```

Comment:
@danielmwakanema I am not entirely clear why we need this? What happens without that function?